conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,14 @@
+# --- !Ups
+
+ALTER TABLE `users`
+ADD COLUMN `date_created` DATETIME NULL AFTER `last_login`;
+ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;
+
```

Comment:
Hi Ken!

LOL!

Are you talking about after the date_created?

thanks!

Sincerely,
Sam
On Nov 30, 2015 8:53 PM, "Ken Dunlap" <notifications@github.com> wrote:

> In conf/evolutions/default/85.sql
> <https://github.com/FEMR/femr/pull/461#discussion_r46231691>:
>
> > @@ -0,0 +1,14 @@
> > +# --- !Ups
> > +
> > +ALTER TABLE `users`
> > +ADD COLUMN `date_created` DATETIME NULL AFTER `last_login`;
> > +ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;
> > +
>
> I swear I'm not being difficult on purpose :-/ lol
>
> change the end of line 4 to a comma rather than a semicolon
>
> ALTER TABLEusers`
> ADD COLUMN date_created DATETIME NULL AFTER last_login,
> ADD COLUMN created_by INT NOT NULL AFTER date_created;`
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FEMR/femr/pull/461/files#r46231691>.
>